tests/components/roku/test_media_player.py

```diff
@@ -188,22 +178,58 @@ async def test_tv_supported_features(
     )
 
 
-async def test_attributes(hass: HomeAssistantType, requests_mock: Mocker) -> None:
+async def test_attributes(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
     """Test attributes."""
-    await setup_integration(hass, requests_mock)
+    await setup_integration(hass, aioclient_mock)
 
     state = hass.states.get(MAIN_ENTITY_ID)
     assert state.state == STATE_HOME
 
     assert state.attributes.get(ATTR_MEDIA_CONTENT_TYPE) is None
+    assert state.attributes.get(ATTR_APP_ID) is None
+    assert state.attributes.get(ATTR_APP_NAME) == "Roku"
+    assert state.attributes.get(ATTR_INPUT_SOURCE) == "Roku"
+
+
+async def test_attributes_app(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
+    """Test attributes for app."""
+    await setup_integration(hass, aioclient_mock, app="netflix")
+
+    state = hass.states.get(MAIN_ENTITY_ID)
+    assert state.state == STATE_PLAYING
+
+    assert state.attributes.get(ATTR_MEDIA_CONTENT_TYPE) == MEDIA_TYPE_APP
+    assert state.attributes.get(ATTR_APP_ID) == "12"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)